src/lib/rest/httpRequestSend.cpp

```diff
@@ -708,7 +715,7 @@ int httpRequestSend
                                      orig_content_type,
                                      content,
                                      fiwareCorrelation,
-                                     ngisv2AttrFormat,
```

Comment:
Is this an actual change? Maybe another TAB should be replace by space indents here...

Reply:
Yes it is. Note old ngis vs fixed one ngsi

Reply:
Ah, ok, I didn't realized on the typo in the original code. Nice :)

NTC